js/profile-w3c-common.js

```diff
@@ -76,9 +75,9 @@ define(
     /*Linter must be the last thing to run*/
     "core/linter",
   ],
-  (domReady, runner, { ui }, ...plugins) => {
+  (runner, { ui }, ...plugins) => {
```

Comment:
I think we should just make this function `async` and inline the domReady() function here. That will clean up this quite a bit.

Reply:
As I said, it didn't work. I completely have no idea why... Both Edge and Firefox didn't even try to execute the callback if I use async function.

Reply:
This is what I tried:

```js
  async (runner, { ui }, ...plugins) => {
    ui.show();
    await domReady();
    try {
      await runner.runAll(plugins);
      await document.respecIsReady;
    } catch (err) {
      console.error(err);
    } finally {
      ui.enable();
    }
  }
```

Reply:
Found the cause: https://github.com/requirejs/requirejs/pull/1729#issuecomment-382297882

OMG, seriously.

Reply:
Argh! sorry you had to go through that.